front-end/src/components/recipelist/recipelist.css

```diff
@@ -0,0 +1,62 @@
+body {
+    font-family: Arial, sans-serif;
+    background-color: #f8f8f8;
+    color: #333;
+    margin: 0;
```

Comment:
Did we all agree on the styling/color scheme? I think we should discuss this with the other teams. Like the UI diagrams are all in pink, is that our final color scheme?

Reply:
No, this is just initial. We can change it later.